draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -871,8 +895,16 @@ handshake is as follows:
 * Host B computes the HMAC of the DCCP-Request and sends a DCCP-Response
   with Confirm feature option for MP-Capable and the MP_JOIN option with
   the Token TB and a random nonce RB together with the computed MP_HMAC.
+  The HMAC is calculated by taking the leftmost 20 bytes from the SHA1 hash
```

Comment:
MPTCP RFC points in https://datatracker.ietf.org/doc/html/rfc8684#section-3.2 for HMAC defintion to RFC2104 and 6234
```
This specification defines
   that HMAC as defined in [RFC2104] is used, along with the SHA-256
   hash algorithm [RFC6234]
```

Would this make sense four our draft as well?

Reply:
Beware that the RFC you linked is for MPTCP v1 which uses SHA256, while in our draft we use SHA1 like in MPTCP v0 (see RFC6824). It makes sense to add the same references, but it's also worth considering if we should move to SHA256 as well

Reply:
Propose to use the approach from MPTCP v1